Soccar [UI]/src/nl/soccar/ui/fx/controller/GameResultsFXMLController.java

```diff
@@ -87,41 +91,44 @@ public void initialize(URL url, ResourceBundle rb) {
     }
 
     /**
-     * Method that display the current settings of the room on the session view.
+     * Method that displays a summary of the game on the GameResults scene.
      */
     public void setGameInfo() {
-        Room room = currentSession.getRoom();
-        Game game = currentSession.getGame();
-
-        for(Player p : room.getTeamBlue().getPlayers()) {
-            int goals = 0;
-            ResultTableItem r = new ResultTableItem(p.getUsername(), goals);
+        room = currentSession.getRoom();
+        game = currentSession.getGame();
+        
+        for (Player p : room.getTeamBlue().getPlayers()) {       
+            ResultTableItem r = new ResultTableItem(p.getUsername(), countGoalsUser(EventType.GOAL_BLUE, p));
             tblBlueGoalsList.getItems().add(r);
         }
-      
-        for(Player p : room.getTeamRed().getPlayers()) {
-            int goals = 0;
-            ResultTableItem r = new ResultTableItem(p.getUsername(), goals);
+        
+        for (Player p : room.getTeamRed().getPlayers()) {
+            ResultTableItem r = new ResultTableItem(p.getUsername(), countGoalsUser(EventType.GOAL_RED, p));
             tblRedGoalsList.getItems().add(r);
         }
-        //TODO: count goals per username.
         
-        lblRoomName.setText("Game Results - " + room.getName());
-        //TODO: Set endscore
+        StringBuilder scoreString = new StringBuilder();
+        scoreString.append("RED ");
+        scoreString.append(game.getEventCountByType(EventType.GOAL_RED));
+        scoreString.append(" - ");
+        scoreString.append(game.getEventCountByType(EventType.GOAL_BLUE));
+        scoreString.append(" BLUE");
         
+        lblScoreEnd.setText(scoreString.toString());
+        lblRoomName.setText("Game Results - " + room.getName());
+    }
+    
+    private int countGoalsUser(EventType type, Player p) {

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused method parameter "type". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AS1172)